### PR TITLE
chore(rust/scanner): improve naming

### PR DESCRIPTION
**Pull request by hyf0**

### CODE REVIEW

**Comment by hyf0** [comment]
* **#1818** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1818?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1816** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1816?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1815** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1815?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1814** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1814?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1813** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1813?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hyf0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/rolldown/src/ast_scanner/mod.rs [hunk 1]
```diff
@@ -516,13 +516,19 @@ impl<'me> AstScanner<'me> {
...
-      Some(symbol_id) if self.is_top_level(symbol_id) => Some(symbol_id),
+      Some(symbol_id) => {
+        if self.is_top_level(symbol_id) {
+          Some(symbol_id)
+        } else {
+          None
+        }
+      }
       None => {
         if ident.name == "module" {
```
Comment (hyf0) [inline crates/rolldown/src/ast_scanner/mod.rs]: This is wrong. User-defined `moduel` var should not be catch here.

**Comment by IWANABETHATGUY** [comment]
Can you add a related test case or link a related issue, since your pr title starts with `fix`?

**Comment by hyf0** [comment]
> Can you add a related test case or link a related issue, since your pr title starts with `fix`? 

👌

**Comment by hyf0** [comment]
Old behavior is correct. I reverted related code.

**Comment by hyf0** [comment]
Change to more suitable title.

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**